session.md

```diff
@@ -56,6 +56,8 @@ php artisan make:session-table
 php artisan migrate
 ```
 
+The `mongodb` connection provided by the `mongodb/laravel-mongodb` package is compatible with the `database` session driver, but using the `mongodb` cache driver with a `mongodb` connection is more optimized for cache expiration and storage.
```

Comment:
I'm a bit confused by the naming conventions here. How is `mongodb` the database driver differentiated from `mongodb` the cache driver?

Reply:
Could we rephrase this sentence? It needs quite a bit of mental gymnastics to understand in the current form. Something like:
The `mongodb` connection provided by the `mongodb/laravel-mongodb` package is compatible with the `database` session driver. However, it's recommended to use the `mongodb` cache driver for improved cache management, especially for handling expiration and storage efficiently.